cwl/src/main/scala/cwl/CommandLineTool.scala

```diff
@@ -167,8 +168,9 @@ case class CommandLineTool private(
 
     // Parse content as json and return output values for each output port
     def parseContent(content: String): EvaluatedOutputs = {
+      val yaml = YamlUtils.parse(content)
       for {
-        parsed <- io.circe.yaml.parser.parse(content).flatMap(_.as[Map[String, Json]]).leftMap(error => NonEmptyList.one(error.getMessage))
+        parsed <- yaml.flatMap(_.as[Map[String, Json]]).leftMap(error => NonEmptyList.one(error.getMessage))
```

Comment:
Can we do better here? We know that `error.getMessage` is just going to return `"null"` - which still doesn't really help. 

Do we know anything about the scenarios which lead to the match errors? eg something like `NonEmptyList.one("Your yaml input didnt have any content")`?

Reply:
Interesting question. I don't have a quick answer.

I haven't looked into what parsed inputs return `null`. There may be other cases besides empty files that return `null`, say just a bunch of comments like:

```yaml
# Do comments count as content YAML?
```

This PR is focused on protecting the server from users causing a DOS, not protecting users from themselves, nor our us supporting the platform from users. That's one reason I tried to emulate as much as possible the original `circe-yaml` error messages.